webdriver/tests/classic/perform_actions/wheel.py

```diff
@@ -56,6 +56,15 @@ def test_scroll_iframe(session, test_actions_scroll_page, wheel_chain):
     wheel_chain.scroll(0, 0, 5, 10, origin=target).perform()
 
     events = get_events(session)
+
+    timeout_start = time.time()
+    # Chrome requires some time (~10ms) to process the event from the iframe, so try to get events until they are in
+    # place, for not longer than for 500ms.
+    while time.time() < timeout_start + 0.5:
```

Comment:
@sadym-chromium is there a reason why we used AsyncPoll for BiDi but not Poll for classic?

Reply:
Pure lack of research of how it is done in other places from my side. https://github.com/web-platform-tests/wpt/pull/46336